test/tests/performance/pbs_jobperf.py

```diff
@@ -56,7 +56,7 @@ def set_test_config(self, config):
         Sets test level configuration
         """
         testconfig = {}
-        for key, value in config.iteritems():
+        for key, value in config.items():
```

Comment:
do we need to list()?

Reply:
No. https://docs.python.org/3/tutorial/datastructures.html#looping-techniques